DigitalLearningSolutions.Web/Views/LearningPortal/Current/CurrentCourseCard/_CurrentCourseCard.cshtml

```diff
@@ -1,11 +1,11 @@
 ﻿@using DigitalLearningSolutions.Web.ViewModels.LearningPortal.Current
 @model CurrentCourseViewModel
 
-<div class="searchable-element nhsuk-panel @Model.DateStyle()" id="@Model.Id-card">
+<div class="searchable-element nhsuk-panel @Model.DateStyle()" id="@Model.Id-course-card">

```

Comment:
I noticed that the current course card and self assessment card used the same Id template ("{Id}-card") which  could conflict on the off chance the id of a self assessment matched the id of a course (which could be possible given they are at different tables). So I took the liberty to change these here.